packages/codemod/transformations/core/v2-to-v3/next-imports-migration.ts

```diff
@@ -0,0 +1,16 @@
+import { CORE_IMPORT_PATH } from "../../../src/consts";
```

Comment:
```suggestion
import { CORE_NEXT_IMPORT_PATH } from "../../../src/consts";
```

Reply:
It should be renamed to `monday-ui-react-core` (which be later on changes to `@vibe/core`)

Reply:
oh right